ci/clair-scan/run.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -eo pipefail
+
+ostype=""
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    ostype="linux"
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+    ostype="darwin"
+else
+    echo "Unsupported OS type $OSTYPE"
+    exit 1
+fi
+
+if [ -n "$1" ]; then
+    REPORTS_OUT_DIR=$(cd "$1" && pwd)
+fi
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+pushd "$THIS_DIR" > /dev/null
+
+# The highest Theia release version, excluding release candidates.
+# We also exclude peeled tags from the output (vX.Y.Z^{}) as they could lead to
+# an invalid version number.
+VERSION=$(git ls-remote --tags --ref https://github.com/antrea-io/theia.git | \
+              grep -v rc | \
+              awk '{print $2}' | awk -F/ '{print $3}' | \
+              sort --version-sort -r | head -n 1)
+
+echo "Scanning Theia version $VERSION"
+
+docker pull "antrea/theia-clickhouse-monitor:$VERSION"
+docker pull "antrea/theia-clickhouse-monitor:latest"
+docker pull "antrea/theia-clickhouse-server:21.11"
+docker pull "antrea/theia-metrics-exporter:0.18.2"
+docker pull "antrea/theia-grafana:8.3.3"
+docker pull "antrea/theia-clickhouse-operator:0.18.2"
```

Comment:
Maybe improve in a follow-up patch:

I suggest we add a deps/ directory and put the third-party dependency versions there. An example: https://github.com/antrea-io/antrea/tree/main/build/images/deps .

Then CI script can `clickhouse_version="$(cat $THIS_DIR/../../build/image/clickhouse)`, and then `docker pull antrea/theia-clickhouse-server:$clickhouse_version`. I think in future we may have more and more script or Dockerfile which will rely on this mechanism, it's easier to maintain if we track the versions in one place.

Reply:
Thanks for the suggestion. I will open a follow-up for that.